firewalld/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Would it be too large to use fedora as base?
+FROM registry.fedoraproject.org/fedora:latest
+
+ENV VERSION=0 RELEASE=1 ARCH=X86_64
+
+LABEL com.redhat.component="firewalld" \
+      name="$FGC/firewalld" \
+      version="$VERSION" \
+      release="$RELEASE.$DISTTAG" \
+      architecture="$ARCH" \
+      usage="atomic install --system --system-package=no firewalld && systemctl start firewalld" \
+      summary="The firewalld as a system container." \
+      maintainer="Rui xin Bao <rubao@redhat.com>" \
+      atomic.type="system"
+
+RUN dnf install --setopt=tsflags=nodocs -y firewalld && \
+    rpm -V firewalld && \
+    dnf clean all && \
+    mkdir -p /exports/hostfs/var/log/ && \
+    mkdir -p /exports/hostfs/etc/{sysconfig,firewalld} && \
+    mkdir -p /exports/hostfs/usr/share/polkit-1/actions && \
+    mkdir -p /exports/hostfs/etc/dbus-1/system.d && \
+    cp /etc/sysconfig/firewalld /exports/hostfs/etc/sysconfig && \
+    # Copy all the polkit related files into current polkit
+    cp /usr/share/polkit-1/actions/org.fedoraproject.FirewallD1.* /exports/hostfs/usr/share/polkit-1/actions && \
```

Comment:
Good point :D. 
yea ...., I think it would be hard to do so on Atomic Host to make a change to read only /usr .. IMO, the way how package layering works in Atomic Host is that we make a new bootable deployment which contains the completely new /usr with the package installed ( package layering avoided modifying current /usr directly ). So in this case, the polkit related files will still be in /usr/share, except that it is inside a deployment folder..
```
After package layering firewalld on atomic host, the location of policy is: 
/sysroot/ostree/deploy/fedora-atomic/deploy/c3d2bde48ccd610057a7d7bb8dbb883bc23a86b61fbef74c934e14253ba7f475.0/usr/share/polkit-1/actions/org.fedoraproject.FirewallD1.policy
```
So, I think what we could possibly do on atomic host is that.. 
1: we first check if we are on an atomic host or an ostree managed OS? 
2: If we are,  we find out where the current bootable deployment is (somehow), then we directly copy these files into that bootable deployment /usr directory. would that work? WDYT @giuseppe

Reply:
is it possible to configure polkit to look into another directory?  We can probably change it to use `/var/` or `/etc` instead of `/usr/share`.  That will work on Atomic Host without the need of copying files to `/usr` and breaking the assumption it is read only.